app/src/main/res/values/strings.xml

```diff
@@ -61,6 +62,7 @@
   <string name="provider_modifications">Modifications</string>
   <string name="menu_upload_single">Upload</string>
   <string name="categories_search_text_hint">Search categories</string>
+  <string name="depicts_search_text_hint">Search depicts</string>
```

Comment:
Please replace with `Search for items that your media depicts (mountain, Taj Mahal, etc.)`
Thanks! :-)